MapboxVisionTests/ObservableVideoSourceTests.swift

```diff
@@ -0,0 +1,130 @@
+@testable import MapboxVision
+import XCTest
+
+private enum Constants {
+    static let numberOfConcurrentMethodCalls = 10000
+    static let expectedTestTimeout = 5.0
+}
+
+class ObservableVideoSourceTests: XCTestCase {
+    private var observableVideoSource: ObservableVideoSource!
+    private var callsCounter: Int!
+
+    private var testExpectation: XCTestExpectation!
+
+    private var concurrentQueue = DispatchQueue(label: "com.mapbox.MapboxVision.ObservableVideoSourceTests.concurrentQueue",
+                                                qos: .default,
+                                                attributes: .concurrent)
+    private var serialQueue = DispatchQueue(label: "com.mapbox.MapboxVision.ObservableVideoSourceTests.serialQueue",
+                                            qos: .default,
+                                            attributes: [])
```

Comment:
let's remove `attributes` since the constructor has default value

Reply:
Agree, updated.